src/screen/GameScreen.java

```diff
@@ -812,7 +812,7 @@ && checkCollision(bullet, enemyShip)) {
 						isExecuted = false;
 						recyclable.add(bullet);
 
-						if (itemManager.dropItem()) {
+						if (itemManager.dropItem() && enemyShip.getHealth() < 0) {
```

Comment:
How about changing like this? I think it would be logically better that call dropItem when enemyShip's health below 0.

```suggestion
						if (enemyShip.getHealth() < 0 && itemManager.dropItem()) {
```

Reply:
Requirements were reflected.